lib/index.js

```diff
@@ -188,15 +188,24 @@ exports.redirect = (ctx, state, data, redirectUrl, nextConfigurationState) => {
     return { status: 302, headers: { location } };
 };
 
+function generateIssuerSubject(ctx) {
+    return {
+        issuerId: `uri:fusebit-template:${ctx.functionId}:${ctx.body.subscriptionId}:${ctx.body.boundaryId}:${ctx.body.functionId}`,
+        subject: 'client-1',
+    };
+}
+
 exports.createStorage = async (ctx) => {
     let issuerCreated = false;
     let issuerId;
     let clientId;
 
     try {
+        let subject;
+        ({ issuerId, subject } = generateIssuerSubject(ctx));
```

Comment:
The two lines above can be written as `let { issuerId, subject } = generateIssuerSubject(ctx);`

Reply:
I think you can lift it out of the `try` and do it this ^ way.

Reply:
@tjanczuk `issuerId` is needed outside of the `try` block

Reply:
Ok, then `subject` will be available at a higher scope without being used, but whatever